sod/spells.js

```diff
@@ -0,0 +1,1163 @@
+let DEBUGMODE = false;
```

Comment:
I think the end state should be to use [js modules](https://caniuse.com/es6-module), and then we can have `sod/spells` import `era/spells` and mutate/add things on top.